text/0006-architecture-review-process.md

```diff
@@ -98,17 +98,17 @@ The key outcome for middle management is to understand the value proposition and
 
 The second two considerations shape how the review process actually happens.
 
-The chair and the team frame the question or questions at the start of the process to ensure agreement on what should be tested. The chair is accountable for answering the question or questions following the review process.
+The chair and the team frame the proposal at the start of the process to ensure agreement on what should be tested. The chair is accountable for summarizing the response to the proposal following the review process.
 
-In order to agree on the question(s), the chair and the team/technical lead or principals will generally have an initial meeting. This meeting should be as small as possible and is intended to:
+In order to agree on the proposal, the chair and the team/technical lead or principals will generally have an initial meeting. This meeting should be as small as possible and is intended to:
 
 * Establish vocabulary
 * Help the team "argue the right level" for the review
 * Identify stakeholders and domain experts — those Consulted, per RACI — in the organization
 
 Subsequently, the team prepares a review packet detailing the problem space and their approach.
 
-The goal is to have the chair (representing the responsible decision makers) and the team establish some shared reality informally, and then to have the team document this and explain how their work addresses the decision maker’s question within that reality.
+The goal is to have the chair and the team establish some shared reality informally, and then to have the team document this and explain how their work addresses problem.
```

Comment:
Is the shared reality informal, is it informally shared, or is it informally established?

s/work addresses problem/proposal would address the problem